pkg/web_app/lib/src/widget/downloads_chart/computations.dart

```diff
@@ -41,3 +41,40 @@ Iterable<String> prepareRanges(List<VersionRangeCount> rangeDownloads) {
 
   return (ranges: ranges, weekLists: result.reversed.toList());
 }
+
+/// Calculates closest point from [point] to line defined by [startPoint] and
```

Comment:
Is this more precise?
```suggestion
/// Calculates closest point to [point] on the line segment between[startPoint] and
```

Reply:
I updated the comment a to something more similiar.